.github/workflows/on-pr.yml

```diff
@@ -0,0 +1,41 @@
+name: "Validate and plan"
+
+on:
+  pull_request:
+
+jobs:
+  test:
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        java-version: [ 8 ]
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-java@v2
+        with:
+          distribution: 'adopt'
```

Comment:
Other workflows here are using temurin. Why adopt? One way or the other, we should be consistent. I haven't kept up with the different JDKs since they split.